api/services/get-user-details.ts

```diff
@@ -0,0 +1,11 @@
+import {AxiosResponse} from 'axios';
```

Comment:
why not call this idam.ts and so we can put all functions that relate to idam here